### ISSUE TITLE
Creating a Group using powershell makes creater a member too

### ISSUE THREAD

**Issue by asajrawat211993** [MicrosoftDocs/office-docs-powershell#3943]
I am using the following commands to create a group and add a member as owner to group;
`
 New-UnifiedGroup -AccessType Public -Alias $groupIdentity -DisplayName $groupDisplayName
 Add-UnifiedGroupLinks -Identity $groupIdentity -LinkType Members -links $smtpAddress
 Add-UnifiedGroupLinks -Identity $groupIdentity -LinkType Owners -Links $smtpAddress
`
I see after doing above the group contains the member $smtpAddress as owner(This is what I am trying to do), in addition to that group has the admin smtp address as owner and member too(admin credentials using which, I logged into O365 exchange, msonline and azuread), Is there a way I can create a group with only specified members as Group members using powershell?

**Comment by tseward** [comment]
Yes, this is by design. An Owner is both an Owner and a Member of a Group. This way they have access to all services, e.g. Planner.

**Comment by asajrawat211993** [comment]
> Yes, this is by design. An Owner is both an Owner and a Member of a Group. This way they have access to all services, e.g. Planner.

@tseward , Yeah that is right, what I am asking is totally different thing, Let me rephrase it. 
The credentials I use to logging into azureAd,exchangeonline and msonline so that I can execute those commands, I see that those credentials username is also part of the group.  I don't want that to  happen, is there any workaround for that?

**Comment by yogkumgit** [comment]
@officedocsbot assign @yogkumgit

**Comment by dariomws306** [comment]
Hi @asajrawat211993,

When you create a group it should have an owner. If you don't specify one when you run New-UnifiedGroup, the user who create the group will be the default owner.

Try using -Owner parameter:
New-UnifiedGroup -AccessType Public -Alias $groupIdentity -DisplayName $groupDisplayName -Owner $groupOwner

**Comment by yogkumgit** [comment]
@dariomws306 Thank you very much for the contribution and digging deep into it. @asajrawat211993 If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Please keep contributing with your suggestions to improve the documentation.
